core/src/labels/labels.cpp

```diff
@@ -217,23 +221,19 @@ void Labels::skipTransitions(const std::vector<std::unique_ptr<Style>>& _styles,
         TileID tileID = tile->getID();
         std::shared_ptr<Tile> proxy;
 
+        const auto& source = _scene->getTileSource(tile->sourceID());
+        if (!source) { continue; }
+
```

Comment:
no, ClientDataSources are not in Scene TileSources so this needs to be handled differently